feature_engine/variable_manipulation.py

```diff
@@ -2,33 +2,77 @@
 # License: BSD 3 clause
 # functions shared across transformers
 
+from typing import List, Optional, Union
 
-def _define_variables(variables):
-    # Check that variable names are passed in a list.
-    # Can take None as value
+import pandas as pd
+
+
+def _define_variables(variables: Union[None, List[str]]) -> Union[List[str], None]:
+    """
+    Takes string or list of strings and checks if argument is list of strings.
+    Can take None as argument.
+
+    Args:
+        variables: string or list of strings
+
+    Returns:
+        List of strings
+    """
+
+    #! Can take Tuple of variables and return list of tuple. Is it accaptable?
     if not variables or isinstance(variables, list):
```

Comment:
No, actually that is not acceptable. Thanks for picking it up. Could you introduce a fix?

Reply:
I fixed this bug and will send PR in develop branch.